ProductsWPF/Categories.xaml.cs

```diff
@@ -0,0 +1,85 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using AccessServices;
+using AccessServices.DTOs;
+using AccessServices.Infrastructure;
+using Ninject;
+using System.Reflection;
+using AccessServices.Interfaces;
+
+namespace ProductsWPF
```

Comment:
It is accepted, that projects name with the following format, '{AppName}.{Layer}'.
 ProjectName.DataAcess, ProjectName.Domain, ProjectName.Service, ProjectName.Design (for WPF windows) for example